src/style.css

```diff
@@ -0,0 +1,144 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 400;
+  font-style: normal;
+  src: url(fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  font-style: medium;
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: 700;
+  font-style: bold;
+  src: url(fonts/Roboto-Bold.ttf);
+}
+
+body {
+  margin: 0;
+  font-family: roboto, sans-serif;
```

Comment:
![2023-04-20_15h41_00](https://user-images.githubusercontent.com/61904995/233368933-cbd0fea3-32c7-432e-bc3e-8bc0b4128d2f.png)
```suggestion
@font-face {
  font-family: Roboto;
  font-weight: 700;
  font-style: bold;
  src: url(fonts/Roboto-Bold.ttf);
}

body {
  margin: 0;
  font-family: Roboto, sans-serif;
```